docs/source/user_guides/superset_guide.rst

```diff
@@ -0,0 +1,149 @@
+:orphan:
+
+Superset guide
+---------------------
+
+Description
+~~~~~~~~~~~
+
+| The Dialog Flow Stats module can be used to obtain and visualize usage statistics for your service.
+| Data aggregation relies on the `OpenTelemetry protocol <#>`_ and the `OpenTelemetry collector <#>`_ along with `Clickhouse <https://clickhouse.com/>`_ as an OLAP storage.
+| Interactive visualization is powered by `Apache Superset <https://superset.apache.org/>`_.
+| All the mentioned services are shipped as Docker containers, including a pre-built Superset image that ensures API compatibility.
+| Authorization credentials can be automatically configured through environment variables.
+
+.. code-block:: shell
+
+    echo 'SUPERSET_USERNAME=...' >> .env
+    echo 'SUPERSET_PASSWORD=...' >> .env
+    docker run --env-file='.env' ghcr.io/deeppavlov/superset_df_dashboard:latest
+
+Collection procedure
+~~~~~~~~~~~~~~~~~~~~
+
+**Installation**
+
+.. code-block:: shell
+
+    pip install dff[stats,clickhouse]
+    pip install dff[stats,postgresql]
```

Comment:
Also, installing `clickhouse` and `postgresql` seems unnecessary. `clickhouse` contains requirements for testing clickhouse, which user doesn't need, and `postgresql` contains requirements for context storage which is not used here.